package-lock.json

```diff
@@ -111,7 +111,7 @@
         "style-loader": "0.23.1",
         "styled-components": "5.3.9",
         "terser": "5.16.2",
-        "trezor-connect": "8.2.6-extended",
+        "trezor-connect": "8.2.12",
```

Comment:
@kdenhartog you need to run `npm install` after changing package.json, otherwise package-lock.json still has the wrong version.

Reply:
(and commit the resulting changes)